commands/Moderation/mute.js

```diff
@@ -0,0 +1,65 @@
+/*
+
+	To use this correctly, you will also need the unmute task located in
+	/tasks/unmute.js
+
+*/
+const { Command, Duration } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+
+			description: 'Mutes a mentioned user.',
+			usage: '<temp|run:default> [when:time|days:number] <member:user> [reason:string] [...]',
```

Comment:
the user argument doesn't make sense, you want a member you use the member argument as it already fetches the member.

Reply:
This is due to already established conventions in ban and softban

Reply:
Oh I'm aware, but as I stated, already established conventions.

Reply:
just because it is done in another command doesn't mean it cannot be different here.